src/test/kotlin/com/igorwojda/integer/pyramidgenerator/challenge.kt

```diff
@@ -4,7 +4,23 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generatePyramid(n: Int): List<String> {
-    TODO("not implemented")
+    val listOfHash = mutableListOf<String>()
+    val maxHash = n * 2 - 1
+    for (i in 1..maxHash step 2) {
```

Comment:
Can you think of a way of constructing the result list while iterating instead of appending to a mutable list?